README.md

```diff
@@ -22,7 +23,18 @@ The Amber module consists of two components:
 * An **Nginx module** that identifies pages to be cached, schedules them for caching, and links to cached pages
 * A **caching script** that runs periodically to cache new pages and check on the status of existing pages
 
+#### Representations ####
+This documentation contains the following representations:
+* **$WEBROOT** - We represent where the root of your homepage is as $WEBROOT. (For example, in our install our webroot was /usr/local/nginx/html)
+* **$NGINXCONFDIR** -We also represent the Nginx configuration directory as $NGINXCONFDIR. (For example, in our install our Nginx configuration directory was /usr/local/nginx)
+* **$BUILDDIR** -We represent the build directory as $BUILDDIR. (For example, in our install our build directory was /usr/local/src)
+* **$DATADIR** -We represent the data directory as $DATADIR. (For example, in our install our data directory was /var/lib)
+* **$LOGDIR** -We represent the log directory as $LOGDIR. (For example, in our install our log directory was /var/log)
+* **$WEBROLE** -We run all services (nginx, FastCGI, database file owner, & cron), as a single user. We represent this user as $WEBROLE. (For example, in our install our web role user was www-data)
 
+Note that the Amber module reserves the /amber top-level directory. If you already have a top-level directory called Amber, you must modify the instructions.
+
```

Comment:
Might want to clarify that this refers to the top level directory from the perspective of the web server URLs, not the top level directory on the server.